config/deploy.rb

```diff
@@ -0,0 +1,43 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.10.1"
+
+set :application, "hackernews_clone"
+set :repo_url, "https://github.com/gingin77/hackernews_clone"
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+set :branch, 'master'
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, "/home/ginnie/hackernews_clone"
+
+set :migration_role, :app
+
+# Default value for :format is :airbrussh.
+# set :format, :airbrussh
+
+# You can configure the Airbrussh format using :format_options.
+# These are the defaults.
+# set :format_options, command_output: true, log_file: "log/capistrano.log", color: :auto, truncate: :auto
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+append :linked_files, "config/database.yml", "config/secrets.yml"
+
+# Default value for linked_dirs is []
+# append :linked_dirs, "log", "tmp/pids", "tmp/cache", "tmp/sockets", "public/system"
+append :linked_dirs, "log", "tmp/pids", "tmp/cache", "tmp/sockets", "vendor/bundle", "public/system", "public/uploads"
```

Comment:
probably don't need `public/uploads` since this app doesn't deal with file uploads AFAIK

Reply:
same goes for `public/system`. I _think_ that's Paperclip's default dir.

Reply:
might be worth adding `node_modules` if that's a thing on this project

Reply:
Ok - that all makes sense, except I don't use node_modules at all right now so no need to add that. I don't have a `bundle` dir in the `vendor` dir, so I'm thinking that should be removed as well, right?

Reply:
Looks like the default install path for Bundler is `.bundle`, so that should be included in the `linked_dirs` here. Can read more about that over at the [`Capistrano::Bundler` project](https://github.com/capistrano/bundler), which is a dependency of capistrano-rails.